### DISCUSSION TITLE
How should Numeric filter work

### DISCUSSION THREAD

**Discussion by candcconsulting** [iTwin/iTwinUI#792] [General]
I have a numeric column which seems to sort OK
![image](https://user-images.githubusercontent.com/19268085/198817850-f0b045ad-6ec1-4ca6-8e52-92c4a3740465.png)
However when I apply the numeric filter
![image](https://user-images.githubusercontent.com/19268085/198817870-b6b653b5-7f2e-45f5-854f-bed1d0777a96.png)
fieldType='number'
No results are returned
If I put a sortType='number' no results returned ... the Storybook does not mention a filterType, but adding this also returns 0 results

How should I configure the column to return numeric range
                    Filter: tableFilters.NumberRangeFilter(),
However, I note in the Component Tree that DefaultCell seems to be adding filterTypes to filterDate
![image](https://user-images.githubusercontent.com/19268085/198818811-b80c6c09-7d6c-4605-bdd9-27ddc4c8a2da.png)
or is that a red herring
What should be set to enable filtering on real numbers

**Comment by mayank99** [comment]
Can you post a codesandbox showing where `fieldType: 'number'` doesn't work? It's hard to help just looking at screenshots.
